kolibri/plugins/coach/assets/src/modules/classSummary/dataHelpers.js

```diff
@@ -77,8 +77,18 @@ export default {
    */
   getRecipientNamesForExam(state) {
     return function(exam) {
+      const adHocLearners = this.getAdHocLearners(exam.assignments).map(
+        learnerId => state.learnerMap[learnerId].name
+      );
+      const recipientsForGroups =
+        Boolean(exam.groups.length) || Boolean(!adHocLearners.length)
```

Comment:
This is totally fine - just a style note because we've talked about this before: In this particular case I would personally omit the `Boolean` cast because the `||` and ternary operator makes the boolean nature of the logic explicit...

```suggestion
        exam.groups.length || !adHocLearners.length
```